crates/router/src/compatibility/stripe/refunds.rs

```diff
@@ -49,6 +49,47 @@ pub async fn refund_create(
     .await
 }
 
+#[instrument(skip_all)]
+#[get("/{refund_id}")]
```

Comment:
Would this have to be `/sync` @Abhicodes-crypto? As far as I remember, trying to register two handlers for the same route with the same HTTP method would cause the Actix server to panic at startup.